card/card.gemspec

```diff
@@ -3,6 +3,7 @@
 require "../decko_gem"
 
 DeckoGem.gem "card" do |s, d|
+  required_ruby_version = ">= 3.0.0"
```

Comment:
DeckoGem handles required_ruby_version

Reply:
Codeclimate doesn't like it if a gem does not specify a ruby version.